### ISSUE TITLE
Alert articles-Unable to find cursor when it is on product images

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#132]
Issue Description:
Unable to see the cursor when focus indicator is on the product images.  This makes difficult for sighted keyboard users to know where cursor is location.  The reason the user knows its on a link is because the link appears in bottom of the browser window.

Impact:  It also affects other notifications/alerts if product images exist.

Location:  https://recalls-rappels.canada.ca/en/alert-recall/rh-illuminated-mirrors-recalled-due-potential-fire-and-shock-hazard
Testing Agent: Chrome v99, FF v97
Both EN and FR

Steps to reproduce:
1.	Go to link above 
2.	Using the keyboard tab to the product images 
3.	Problem:  The cursor disappears when on product images.
![RSAMS-Unknown Cursor placement](https://user-images.githubusercontent.com/88147013/159276617-6fd93682-bdda-4163-becf-df4cd06e5d86.jpg)

Expected Result:
The cursor is required to be visible when navigating throughout the web page, including product images.

**Comment by davidgotlieb** [comment]
Fixed by OP.

**Comment by hydegeoff** [comment]
The seems to be fixed in RSAMS dev, not in Production
